src/seccompiler/Cargo.toml

```diff
@@ -20,7 +20,6 @@ bincode = "1.2.1"
 clap = { version = "4.5.21", features = ["derive", "string"] }
 displaydoc = "0.2.5"
 libc = "0.2.167"
-libseccomp = "0.3.0"
```

Comment:
let's merge this commit into the first one, so we don't add a dependency and then remove it again 2 commits later in the same PR.